conf/distro/fsl-wayland.conf

```diff
@@ -6,7 +6,7 @@ DISTRO = "fsl-wayland"
 DISTRO_NAME = "FSL Wayland"
 
 # Define Init System
-INIT_MANAGER = "systemd"
+INIT_MANAGER = "${@bb.utils.contains('DISTRO_FEATURES', 'sysvinit', 'sysvinit', 'systemd', d)}"
```

Comment:
I believe you should just set `INIT_MANAGER` directly to change to `sysvinit`. Then, the poky conf files will setup many variables, including `DISTRO_FEATURES`. See https://docs.yoctoproject.org/dev/singleindex.html#term-INIT_MANAGER.